slices/erlang-eldap.yaml

```diff
@@ -0,0 +1,17 @@
+package: erlang-eldap
+
+essential:
+  - erlang-eldap_copyright
+
+slices:
+  libs:
+    essential:
+      - erlang-asn1_libs
+      - erlang-base_bins
```

Comment:
why do these libs need the `erland-base_bins`?

Reply:
Those modules are useless on their own and they need the erlang binaries to be run, hence that dependency on erlang-base which contains the binaries.

Reply:
Let's discuss this in https://github.com/canonical/chisel-releases/pull/258#discussion_r1802799227, where there is more context to address your last comment